app/templates/index.html

```diff
@@ -63,6 +63,7 @@ <h4>
     <i class="fa fa-hourglass"></i>
     Loading...
 </div>
+<button class="scrollTopBtn" id="scrollToTopBtn">⇪</button>
```

Comment:
No need to give this a class in addition to an ID; let's just use an ID (and rename it as I mentioned above).
Also, as cool as the icon is, it's a little overcomplicated; can you use ^ instead?